tutorials/R/R-skills/basic-R-skills/R-Basics-Getting-Started/R-Basics-Getting-Started.Rmd

```diff
@@ -657,7 +720,7 @@ x <- factor(c("yes", "no", "yes"), levels = c("yes", "no"))
 x
```

Comment:
Maybe run the `x` in line 718 (and add spacing as needed), above where you re-arrange them, to clearly show the difference in the output.

Reply:
Done